tests_new/integration_tests/modules/s3_datasets/global_conftest.py

```diff
@@ -288,20 +360,112 @@ def persistent_s3_dataset1(client1, group1, persistent_env1, testdata):
-    )
+def persistent_s3_dataset1_tables(client1, persistent_s3_dataset1):
+    return create_tables(client1, persistent_s3_dataset1)
 
 
 @pytest.fixture(scope='session')
-def persistent_imported_kms_s3_dataset1(client1, group1, persistent_env1, testdata):
-    return get_or_create_persistent_s3_dataset(
-        'persistent_imported_kms_s3_dataset1',
-        client1,
-        group1,
-        persistent_env1,
-        'persistentimportedkms',
-        'persistentimportedkms',
-        'persistentimportedkms',
-    )
+def persistent_s3_dataset1_folders(client1, persistent_s3_dataset1):
+    return create_folders(client1, persistent_s3_dataset1)
+
+
+@pytest.fixture(scope='session')
+def persistent_imported_sse_s3_dataset1(client1, group1, persistent_env1, persistent_env1_aws_client, testdata):
+    bucket_name = 'persistentimportedsses3'
+    bucket = None
+    try:
+        s3_client = S3Client(session=persistent_env1_aws_client, region=persistent_env1['region'])
+        bucket = s3_client.bucket_exists(bucket_name)
+        if not bucket:
+            bucket = s3_client.create_bucket(bucket_name=bucket_name, kms_key_arn=None)
+        if not bucket:
+            raise Exception('Error creating import dataset AWS resources for persistent_imported_sse_s3_dataset1')
+        return get_or_create_persistent_s3_dataset(
+            'persistent_imported_sse_s3_dataset1', client1, group1, persistent_env1, bucket_name
+        )
+    except Exception:
+        if bucket:
+            S3Client(session=persistent_env1_aws_client, region=persistent_env1['region']).delete_bucket(bucket)
+
+
+@pytest.fixture(scope='session')
+def persistent_imported_sse_s3_dataset1_tables(client1, persistent_imported_sse_s3_dataset1):
+    return create_tables(client1, persistent_imported_sse_s3_dataset1)
+
+
+@pytest.fixture(scope='session')
+def persistent_imported_sse_s3_dataset1_folders(client1, persistent_imported_sse_s3_dataset1):
+    return create_folders(client1, persistent_imported_sse_s3_dataset1)
+
+
+@pytest.fixture(scope='session')
+def persistent_imported_kms_s3_dataset1(
+    client1, group1, persistent_env1, persistent_env1_aws_client, persistent_env1_integration_role_arn, testdata
+):
+    resource_name = 'persistentimportedkms'
+    bucket = None
+    kms_alias = None
+    database = None
+    existing_lf_admins = None
+    try:
+        # Check and create KMS key
+        kms_client = KMSClient(
+            session=persistent_env1_aws_client,
+            account_id=persistent_env1['AwsAccountId'],
+            region=persistent_env1['region'],
+        )
+        kms_id, kms_alias = kms_client.get_key_id_and_alias(resource_name)
+        if not kms_id:
+            kms_id, kms_alias = kms_client.create_key_with_alias(resource_name)
+        # Check and create S3 Bucket
+        s3_client = S3Client(session=None, region=persistent_env1['region'])
```

Comment:
when session is None, S3Client raises an error, because of 
`    def __init__(self, session, region):
        self._client = session.client('s3', region_name=region)
        self._resource = session.resource('s3', region_name=region)
        self._region = region
`